docs/Dataplex-batchsink.md

```diff
@@ -2,136 +2,141 @@
+
 * Insert - all records will be inserted in destination table.
-* Update - records that match on Table Key will be updated in the table. Records that do not match
-  will be dropped.
+* Update - records that match on Table Key will be updated in the table. Records that do not match will be dropped.
 * Upsert - records that match on Table Key will be updated. Records that do not match will be inserted.
 
 **Table Key**: List of fields that determines relation between tables during Update and Upsert operations.
 
-**Dedupe By**: Column names and sort order used to choose which input record to update/upsert when there are
-multiple input records with the same key. For example, if this is set to 'updated_time desc', then if there are
-multiple input records with the same key, the one with the largest value for 'updated_time' will be applied.
-
-**Partition Filter**: Partition filter that can be used for partition elimination during Update or
-Upsert operations. Should only be used with Update or Upsert operations for tables where
-require partition filter is enabled. For example, if the table is partitioned the Partition Filter
-'_PARTITIONTIME > "2020-01-01" and _PARTITIONTIME < "2020-03-01"',
-the update operation will be performed only in the partitions meeting the criteria.
-
-**Truncate Table**: Whether or not to truncate the table before writing to it.
-Should only be used with the Insert operation.
-
-**Update Table Schema**: Whether the BigQuery table schema should be modified
-when it does not match the schema expected by the pipeline.
-* When this is set to false, any mismatches between the schema expected by the pipeline
-  and the schema in BigQuery will result in pipeline failure.
-* When this is set to true, the schema in BigQuery will be updated to match the schema
-  expected by the pipeline, assuming the schemas are compatible.
-
-**Partitioning Type**: Specifies the partitioning type. Can either be Integer or Time or None. Defaults to Time.
-This value is ignored if the table already exists.
-* When this is set to Time, table will be created with time partitioning.
-* When this is set to Integer, table will be created with range partitioning.
-* When this is set to None, table will be created without time partitioning.
-
-**Range Start**: For integer partitioning, specifies the start of the range. Only used when table doesn’t
-exist already, and partitioning type is set to Integer.
+**Dedupe By**: Column names and sort order used to choose which input record to update/upsert when there are multiple
+input records with the same key. For example, if this is set to 'updated_time desc', then if there are multiple input
+records with the same key, the one with the largest value for 'updated_time' will be applied.
+
+**Partition Filter**: Partition filter that can be used for partition elimination during Update or Upsert operations.
+Should only be used with Update or Upsert operations for tables where
```

Comment:
Delete "Should". "Only use with Update or Upsert...."

Reply:
done